Let's print actual class name length

I got this error:

```text
'DeepInfraRequest' has bad naming, class name is more than 15, consider more simple name
[ERROR] 	read: 'https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html' (LongClassNameCheck)
```

This error only states that 'DeepInfraRequest' class name length is more than `15` chars, but does not print what is the actual length of this class. It can improve developer experience with oop-cop error handling

--- a/src/it/long-class-name/verify.groovy
+++ b/src/it/long-class-name/verify.groovy
@@ -24,6 +24,6 @@
 
 String log = new File(basedir, 'build.log').text;
 [
-  "'ClassNameIsTooComplex' has bad naming, class name is more than 15, consider more simple name\n\tread: 'https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html'"
+  "'ClassNameIsTooComplex' has bad naming, class name is more than 15, the actual length is 21, consider using shorter name\n\tread: 'https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html'"
 ].each { assert log.contains(it): "Log doesn't contain ['$it']" }
 true

--- a/src/main/java/ru/l3r8y/checks/LongClassNameCheck.java
+++ b/src/main/java/ru/l3r8y/checks/LongClassNameCheck.java
@@ -23,6 +23,7 @@
  */
 package ru.l3r8y.checks;
 
+import java.util.Collection;
 import lombok.RequiredArgsConstructor;
 import ru.l3r8y.Check;
 import ru.l3r8y.ClassName;
@@ -30,7 +31,6 @@ import ru.l3r8y.Complaint;
 import ru.l3r8y.complaint.ClassifiedComplaint;
 import ru.l3r8y.complaint.LinkedComplaint;
 import ru.l3r8y.complaint.WrongClassNaming;
-import java.util.Collection;
 
 /**
  * Check for Long class name.

--- a/src/main/java/ru/l3r8y/checks/LongClassNameCheck.java
+++ b/src/main/java/ru/l3r8y/checks/LongClassNameCheck.java
@@ -23,7 +23,6 @@
  */
 package ru.l3r8y.checks;
 
-import java.util.Collection;
 import lombok.RequiredArgsConstructor;
 import ru.l3r8y.Check;
 import ru.l3r8y.ClassName;
@@ -31,6 +30,7 @@ import ru.l3r8y.Complaint;
 import ru.l3r8y.complaint.ClassifiedComplaint;
 import ru.l3r8y.complaint.LinkedComplaint;
 import ru.l3r8y.complaint.WrongClassNaming;
+import java.util.Collection;
 
 /**
  * Check for Long class name.
@@ -59,8 +59,9 @@ public final class LongClassNameCheck implements Check {
                     new WrongClassNaming(
                         this.name,
                         String.format(
-                            "class name is more than %s, consider more simple name",
-                            this.fine
+                            "class name is more than %s, the actual length is %s, consider using shorter name",
+                            this.fine,
+                            this.name.value().length()
                         )
                     ),
                     "https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html"
@@ -78,7 +79,7 @@ public final class LongClassNameCheck implements Check {
      */
 
     /**
-     * Is name ok?
+     * Is the name ok?
      *
      * @return True if longer
      */

